Code exec logs changelog (#168)
vellum-ai/vellum-client-generator: fern/docs/content/change-log/2024-03.mdx

title: Changelog | March, 2024
---

## Code Execution Logs

_March 6th, 2024_

> You can now use print or console.log statements in code execution nodes, you can view the logs by looking at a node's result and clicking the logs tab.

Reviewer: [nit] I would use inline code format, `print` and `console.log`

> You can now use print or console.log statements in code execution nodes, you can view the logs by looking at a node's result and clicking the logs tab.

Reviewer 2: ```suggestion
You can now use print or console.log statements in code execution nodes and view the logs by looking at a node's result and clicking the logs tab.
```

![Code Logs Exec Nodes](https://storage.googleapis.com/vellum-public/help-docs/code-exec-logs.png)

> We have also added logs for eval metrics, you can view them by adding the logs column in the table columns setting.

Reviewer 2: [nit] minor copy changes for consistency
```suggestion
We've also added logs for eval metrics. You can view them by enabling the logs column in the table columns settings.
```
